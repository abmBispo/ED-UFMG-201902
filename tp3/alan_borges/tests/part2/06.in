16
basta de prefacio ao capitulo vamos ao capitulo capitulo lxxx venhamos ao capitulo venhamos ao capitulo
c ao
c capitulo
c vamos
c venhamos
c basta
c de
c lxxx
c prefacio
