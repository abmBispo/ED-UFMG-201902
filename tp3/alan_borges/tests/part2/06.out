01
11
000
001
1000
1001
1010
1011
